content/learning-paths/servers-and-cloud-computing/serverless-framework-aws-intro/lambda.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Deploy AWS Lambda
+weight: 4
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+You are now ready to create a Serverless Framework and deploy Lambda function.
+
+## Create a project
+Open terminal or command prompty and type:
+
+```console
+serverless
```

Comment:
@dawidborycki: The intent I believe is to show serverless framework running on a Windows on Arm machine. 
When you use npm to install and run serverless:

C:\Users\parver01>serverless
Platform windows - arm64 is not supported.